Automattic/woocommerce-payments
# Merchants can change their deposit schedule

Fixes #4464

#### Changes proposed in this Pull Request

With this PR, merchants will be able to change their deposit schedule between Daily, Weekly and, Monthly. When opting for a weekly schedule a desired day of the week can be chosen, with a monthly schedule a day of the month can be chosen.

A new `Deposits` section is added to WCPay Settings where the frequency can be selected.

![image](https://user-images.githubusercontent.com/57298/187828238-b057cdbf-a08a-42af-9cc9-9954c9ee3cd6.png)

![image](https://user-images.githubusercontent.com/57298/187828269-8cd678f1-86d2-44d3-83f5-ffb3ea4accb6.png)

![image](https://user-images.githubusercontent.com/57298/187828295-0e247afd-31c3-4420-862b-c53957a7d87b.png)

If an account is new or deposit scheduling is unavailable we show a warning.

![image](https://user-images.githubusercontent.com/57298/187828559-4a906317-41b1-419c-9184-224905e82243.png)

![image](https://user-images.githubusercontent.com/57298/187828714-671b007c-4c7b-4ed1-96e6-a5e7b43e88e9.png)

#### Testing instructions

Testing of deposits is currently limited to live accounts only. Test accounts will see a warning message. 
Alternatively, with a development version of the Server, testing is available.

1. Visit WC Pay Settings.
2. Change the deposit schedule.
3. Verify the deposit schedule in stripe dashboard.

-------------------


**Post merge**
